### PR TITLE
Feature/search/example graph list

### PR DESCRIPTION
**Pull request by yxyx-github**

### CODE REVIEW

File: src/main/algorithms/search/algorithms/dataHelpers/index.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import type { RawGraphForm, RawGraphFormGrid, RawSearchSimulation } from '@/main/algorithms/search/algorithms/dataHelpers/types'
+import type { SearchSimulation } from '@/main/algorithms/search/algorithms/types'
+import { GraphFormItem } from '@/main/algorithms/search/graphForm/graphFormItem'
+import { GraphForm } from '@/main/algorithms/search/graphForm/graphForm'
+
+// TODO: possible bug: error if start or end item is null
```
Comment (termitehuegel) [inline src/main/algorithms/search/algorithms/dataHelpers/index.ts:6]: remove this comment

File: src/main/algorithms/search/algorithms/dataHelpers/index.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        item.generateItemId() === new GraphFormItem(data.startItem.itemData).generateItemId()
+    ) ?? null)
+    graphForm.setEndItem(graphForm.toItems().find(item =>
+        item.generateItemId() === new GraphFormItem(data.endItem.itemData).generateItemId()
+    ) ?? null)
+
+    return graphForm
+}
+
+// TODO: possible bug: error if start or end item is null
```
Comment (termitehuegel) [inline src/main/algorithms/search/algorithms/dataHelpers/index.ts:19]: remove todo

**Review by termitehuegel — CHANGES_REQUESTED** [review]
please remove the comments. The rest looks good and can approved like this.

File: src/main/algorithms/search/algorithms/dataHelpers/index.ts (hunk 2 above)
Comment (yxyx-github) [reply src/main/algorithms/search/algorithms/dataHelpers/index.ts]: fixed

File: src/main/algorithms/search/algorithms/dataHelpers/index.ts (hunk 1 above)
Comment (yxyx-github) [reply src/main/algorithms/search/algorithms/dataHelpers/index.ts]: fixed

**Review by termitehuegel — APPROVED** [review]
looks good

**Pull request merged**